hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/HalfMergeSortSparkIT.java

```diff
@@ -156,6 +151,7 @@ public void testHalfSortMergeAfterSplit() throws Throwable {
             "(c1 int, c2 int, c3 int, c4 int)");
 
     protected static SpliceIndexWatcher aIndex = new SpliceIndexWatcher(A, CLASS_NAME, A_IDX,CLASS_NAME,"(c1 desc, c2 asc, c3 desc)");
```

Comment:
This test is ignored, and the class is unused, it can be removed.

Reply:
Done.